zuul-core/src/main/java/com/netflix/netty/common/accesslog/AccessLogPublisher.java

```diff
@@ -59,13 +59,15 @@ public void log(
             LocalDateTime dateTime,
             Integer localPort,
             String remoteIp,
+            String originalRemoteIp,
             Long durationNs,
             Integer requestBodySize,
             Integer responseBodySize) {
         StringBuilder sb = new StringBuilder();
 
         String dateTimeStr = dateTime != null ? dateTime.format(DATE_TIME_FORMATTER) : "-----T-:-:-";
         String remoteIpStr = (remoteIp != null && !remoteIp.isEmpty()) ? remoteIp : "-";
```

Comment:
Decided to keep this one as it can provide extra visibility. For example when requests travel through CLBs